android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/FormConfigurationJsonWizardFormActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.vijay.jsonwizard.activities;
+
+import com.vijay.jsonwizard.delegates.JsonWizardFormDelegate;
+
+import org.json.JSONException;
+
+public class FormConfigurationJsonWizardFormActivity extends FormConfigurationJsonFormActivity{
+
+    private final JsonWizardFormDelegate delegate = new JsonWizardFormDelegate();
+
+    @Override
+    public void initializeFormFragment() {
+        initializeFormFragmentCore();
+    }
+
+    @Override
+    public void writeValue(String stepName, String key, String value, String openMrsEntityParent, String openMrsEntity, String openMrsEntityId) throws JSONException {
+        callSuperWriteValue(stepName, key, value, openMrsEntityParent, openMrsEntity, openMrsEntityId);
+    }
+
+    @Override
+    public void onFormFinish() {
+        callSuperFinish();
+    }
+
+    protected void callSuperFinish(){
+        super.onFormFinish();
+    }
+
+    protected void callSuperWriteValue(String stepName, String key, String value, String openMrsEntityParent, String openMrsEntity, String openMrsEntityId) throws JSONException {
+        super.writeValue(stepName, key, value, openMrsEntityParent, openMrsEntity, openMrsEntityId);
+    }
```

Comment:
I thought that these four methods were redundant? What do you think? 

I think that they each call the super and therefore no need to override and only call super with the exactly same params

Reply:
For these I wasn't sure, may be other libraries using native-forms may have had them overridden, because they'd been marked as protected.